src/Tribe/Ticket_Object.php

```diff
@@ -72,6 +72,13 @@ class Tribe__Tickets__Ticket_Object {
 		public $provider_class;
 
 		/**
+		 * Holds the SKU for the ticket
+		 *
+		 * @var string
+		 */
+		public $sku;
```

Comment:
Does this need to be public? Protecting fields and adding an accessor method if needed is usually preferred.